app/presenters/waste_carriers_engine/edit_form_presenter.rb

```diff
@@ -74,8 +74,12 @@ def tier
     private
 
     def format_main_person(person)
-      role = I18n.t("#{LOCALES_KEY}.main_people.#{transient_registration.business_type}")
-      "#{person_name(person)} (#{role})"
+      role = I18n.t("#{LOCALES_KEY}.main_people.#{transient_registration.business_type}", default: "")
+      if role.present?
+        "#{person_name(person)} (#{role})"
+      else
+        person_name(person)
```

Comment:
Seems like codeclimate is complaining about missing test coverage for this `else`, shall we add some?

Reply:
Added an extra test to cover this 👍